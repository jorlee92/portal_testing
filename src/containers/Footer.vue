<template lang="html">
  <footer class="footer">
    <div class="container">
      <div class="content has-text-centered">
    This the footer
      </div>
    </div>
  </footer>
</template>

<script>
export default {
  name: 'pp-footer',
}
</script>

<style lang="sass">
</style>