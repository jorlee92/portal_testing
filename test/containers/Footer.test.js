import { shallow } from 'vue-test-utils'
import Footer from '@/containers/Footer.vue'

describe('Footer', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(Footer)
  })
  it('should render the Footer correctly', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
