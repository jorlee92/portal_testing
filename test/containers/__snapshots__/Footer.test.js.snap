// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render the Footer correctly 1`] = `
<footer
  class="footer"
>
  <div
    class="container"
  >
    <div
      class="content has-text-centered"
    >
      
  This the footer
    
    </div>
  </div>
</footer>
`;
